hathor/manager.py

```diff
@@ -795,14 +795,7 @@ def _make_block_template(self, parent_block: Block, parent_txs: 'ParentTxs', cur
             )
         timestamp = min(max(current_timestamp, timestamp_min), timestamp_max)
         parent_block_metadata = parent_block.get_metadata()
-        # this is the min weight to cause an increase of twice the WEIGHT_TOL, we make sure to generate a template with
-        # at least this weight (note that the user of the API can set its own weight, the block sumit API will also
-        # protect agains a weight that is too small but using WEIGHT_TOL instead of 2*WEIGHT_TOL)
-        min_significant_weight = calculate_min_significant_weight(
-            parent_block_metadata.score,
-            2 * self._settings.WEIGHT_TOL
-        )
-        weight = max(self.daa.calculate_next_weight(parent_block, timestamp), min_significant_weight)
+        weight = self.daa.calculate_next_weight(parent_block, timestamp)
```

Comment:
Keep `calculate_min_significant_weight` for at least one more release. Maybe something like `TODO: remove calculate_min_significant_weight in the future`.

Reply:
Done!